src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -74,17 +80,17 @@
     final HttpEndExFW httpEndExRO = new HttpEndExFW();
     final ListFW<HttpHeaderFW> requestHeadersRO = new HttpBeginExFW().headers();
 
-    public final RouteManager router;
+    final RouteManager router;
+    final Long2ObjectHashMap<Function<HttpBeginExFW, MessageConsumer>> correlations;
     final BudgetManager budgetManager;
+    final HttpProxyCacheableRequestGroup requestGroup;
```

Comment:
Seems like the `requestGroup` would represent a single `requestHash`, so do we want `Int2ObjectHashMap< HttpProxyCacheableRequestGroup>` keyed by `requestHash` then?